README.md

```diff
@@ -65,28 +81,35 @@ last successfully read message is used as the offset id for the subsequent XREAD
 
 ### Record Keys
 
-* Pub/Sub mode: An arbitrary key is created for Pub/Sub messages with the format: `<channel>_<current_ts_nano>`
+* Pub/Sub mode: The redis channel name is uses as the record key
 
-* Stream mode: The message id assigned to the stream message on calling XADD is used to uniquely identify the records in case of Stream mode.
+* Stream mode: The redis key name is used as the record key 
```

Comment:
i.e. all messages from a single Redis stream source have the same key?

Reply:
Yes, As there is no other unique identifier for messages published to channels

Reply:
The stream messages have message_id, and that is used as position. We can use that as a key as well, For now, kept it as a record key for consistency across records in both mode.

Reply:
It feels like key should be "more unique": https://github.com/ConduitIO/conduit-connector-sdk/blob/main/record.go#L34-L37 However, maybe we can handle that in a separate issue.